src/sentry/integrations/slack/actions/message_action.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from collections.abc import Mapping
+from dataclasses import dataclass
+from typing import Any
+
+from sentry.notifications.utils.actions import BaseMessageAction
+
+
+@dataclass
+class SlackMessageAction(BaseMessageAction):
```

Comment:
Moved this over from `BlockKitMessageAction` to be more accurate to Slack specific namespace, and added helper functions so it's easier to enact on the object, and add tests to verify functionality and contracts